### PR TITLE
Niko on paras

### PR DESCRIPTION
**Pull request by Kaivosukeltaja**

### CODE REVIEW

**Comment by JuhQ** [comment]
What is this? Commit message missing and no proper title. I literally have no idea what this is.

image

File: comic.js [hunk 1]
```diff
@@ -1 +1 @@
-document.body.innerHTML += '<style>* { font-family: "Comic Sans MS";} p:nth-child(odd) {color: pink;} p:nth-child(even) {color: red;} li:nth-child(even) {color: orange;}</style>';
\ No newline at end of file
+document.body.innerHTML += '<style>* { font-family: "Comic Sans MS" !important;} p:nth-child(odd) {color: pink;} p:nth-child(even) {color: red;} li:nth-child(even) {color: orange;}</style>';
```
Comment (JuhQ) [inline comic.js]: Pls add !important to other css :shit:

**Comment by Kaivosukeltaja** [comment]
Lolwtf everything is just awesome. Just merge already!

File: comic.js [hunk 2]
```diff
@@ -1 +1 @@
-document.body.innerHTML += '<style>* { font-family: "Comic Sans MS";} p:nth-child(odd) {color: pink;} p:nth-child(even) {color: red;} li:nth-child(even) {color: orange;}</style>';
\ No newline at end of file
+document.body.innerHTML += '<style>* { font-family: "Comic Sans MS" !important;} p:nth-child(odd) {color: pink !important;} p:nth-child(even) {color: red !important;} li:nth-child(even) {color: orange !important;}</style>';
```
Comment (JuhQ) [inline comic.js]: Readability got worse. Pls split code into multiple lines.

File: comic.js [hunk 3]
```diff
@@ -1 +1,6 @@
-document.body.innerHTML += '<style>* { font-family: "Comic Sans MS";} p:nth-child(odd) {color: pink;} p:nth-child(even) {color: red;} li:nth-child(even) {color: orange;}</style>';
\ No newline at end of file
+document.body.innerHTML +=
```
Comment (JuhQ) [inline comic.js:1]: Better but not sure yet. Maybe we should set up a committee to discuss what standard to use for linebreak formatting.

Comment (Kaivosukeltaja) [reply comic.js:1]: WHATEVER

Comment (JuhQ) [reply comic.js:1]: /ban @Kaivosukeltaja

Comment (JuhQ) [reply comic.js:1]: lgtm

**Pull request merged**